src/test/resources/pages/instructorStudentRecordsPageWithScriptInjectionProfile.html

```diff
@@ -39,9 +39,7 @@
                   <td>
                     Stud1&lt;script&gt; alert('hi!'); &lt;/script&gt; (
                     <i>
-                      <span class="text-muted">
-                        Not Specified
-                      </span>
+                      Not Specified
```

Comment:
I think here we should follow to the previous behavior. Replace `(<i> ${student.displayedGenderType} </i>)` in studentProfile.tag (line 24) to something like that:

```jsp
(<i>
<c:choose>
    <c:when test="${student.gender == OTHER or student.gender == null}">
    <span class="text-muted">
        ${student.displayedGenderType}
    </span>
    </c:when>
    <c:otherwise>
        ${student.displayedGenderType}
    </c:otherwise>
</c:choose>
</i>)
```
where OTHER you can specify as `<c:set var="OTHER" value="<%= GenderType.OTHER %>" />`